lib/orbit/ocs/trip.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Orbit.Ocs.Trip do
+  use Ecto.Schema
+  alias Orbit.RailLine
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          service_date: Date.t(),
+          uid: String.t(),
+          train_uid: String.t() | nil,
+          prev_uid: String.t() | nil,
+          next_uid: String.t() | nil,
+          route: String.t() | nil,
+          rail_line: RailLine.t(),
+          trip_type: String.t() | nil,
+          scheduled_departure: DateTime.t() | nil,
+          scheduled_arrival: DateTime.t() | nil,
```

Comment:
Need a field for `offset`?

Reply:
thanks! 746ad71755ca7944f5b9f50d343f2472f316b04d